spec/README.md

```diff
@@ -79,39 +79,41 @@ A reference to a _term_ looks like this.
 > The provisions of the stability policy are not in effect until
 > the conclusion of the technical preview and adoption of this specification.
 
-Updates to this specification will not change
-the syntactical meaning, the runtime output, or other behaviour
-of valid messages written for earlier versions of this specification
-that only use functions defined in this specification.
+Updates to this specification will not make any well-formed or valid _message_ invalid.
```

Comment:
In the refactor, I broke each of the promises apart.

This promise says that, if a message is well-formed, it will still be well-formed in the future and if a message is valid it will still be valid in the future. 

Well-formed is limited to the grammar, i.e. syntax errors. Valid extends to data model errors.

It's plausible to say that we will introduce new, tighter restrictions into the syntax in the future that would result in a message becoming a syntax error that is not currently a syntax error. The extension to well-formed is thus unreasonable to include and I will remove it now.